fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/lao/event/rollcall/RollCallFragment.java

```diff
@@ -252,8 +287,9 @@ private void retrieveAndDisplayPublicKey() {
     PopTokenData data = new PopTokenData(new PublicKey(pk));
     Bitmap myBitmap = QRCode.from(gson.toJson(data)).bitmap();
     binding.rollCallPkQrCode.setImageBitmap(myBitmap);
+    // Set the QR visible only if the rollcall is open and the user isn't the organizer
     binding.rollCallPkQrCode.setVisibility(
-        laoViewModel.isOrganizer() ? View.INVISIBLE : View.VISIBLE);
+        laoViewModel.isOrganizer() || !rollCall.isOpen() ? View.INVISIBLE : View.VISIBLE);
```

Comment:
Just a nitpick: It might be good to add brackets around `laoViewModel.isOrganizer() || !rollCall.isOpen()` as I was confused at first how the logic of this if statement works :)

Reply:
Sure, I'll write that in a cleaner way!